src/MoBi.Presentation/Tasks/Interaction/ParameterValuesTask.cs

```diff
@@ -164,27 +125,26 @@ public override IMoBiCommand RefreshPathAndValueEntitiesFromBuildingBlocks(Param
             ObjectType = ObjectTypes.ParameterValue
          };
 
-         pathAndValueEntitiesToRefresh.Each(pathAndValueEntity =>
-         {
-            var parameter = _parameterResolver.Resolve(pathAndValueEntity.ContainerPath, pathAndValueEntity.Name, spatialStructure, moleculeBuildingBlock);
-            if (parameter == null)
-               return;
-
-            if (!HasEquivalentDimension(pathAndValueEntity, parameter))
-               macroCommand.Add(UpdatePathAndValueEntityDimension(buildingBlock, pathAndValueEntity, parameter.Dimension));
-
-            if (!HasEquivalentPathAndValueEntity(pathAndValueEntity, parameter))
-               macroCommand.Add(SetDisplayValueWithUnit(pathAndValueEntity, parameter.ConvertToDisplayUnit(parameter.Value), parameter.DisplayUnit, buildingBlock));
-
-            // Evaluating the pathAndValueEntity before the formula is important if the pathAndValueEntity is a constant and the original building block uses a constant formula
-            if (!HasEquivalentFormula(pathAndValueEntity, parameter.Formula))
-               macroCommand.Add(ChangeValueFormulaCommand(buildingBlock, pathAndValueEntity, parameter.Formula.IsConstant() ? null : _cloneManagerForBuildingBlock.Clone(parameter.Formula, buildingBlock.FormulaCache)));
-         });
+         // pathAndValueEntitiesToRefresh.Each(pathAndValueEntity =>
+         // {
```

Comment:
let's remove this?

Reply:
or why do we need this?

Reply:
I didn't want to remove code for refresh until we figure out how it's going to work. 

We just need to provide a spatial structure and molecule building block and all this code will work again.

I think that is somewhat the plan right now. Refresh would still work, but you have to pick the building blocks to refresh from.